dags/tasks/amazon/sqs.py

```diff
@@ -10,7 +10,7 @@ def SubscribeOperator(**kwargs) -> SQSSensor:
     queue_name = kwargs.get("queue", "")
     # Defaults to Sinopia dev environment
     sinopia_env = kwargs.get("sinopia_env", "dev")
-    aws_sqs_url = Variable.get(f"sqs_{sinopia_env}")
+    aws_sqs_url = Variable.get(f"SQS_{sinopia_env.upper()}")
```

Comment:
We need to update the [wiki](https://github.com/LD4P/ils-middleware/wiki/AWS-SQS) to reflect this expectation that the Airflow Variable should be upper case, i.e. `SQS_DEV` or `SQS_STAGE`.

Reply:
Updated wiki with capitalized `SQS_DEV` and `SQS_STAGE`.